src/renderer/src/routes/pages/Settings/SettingsNetwork/index.tsx

```diff
@@ -31,41 +31,38 @@ export const SettingsNetwork = () => {
         <h1 className="text-white text-sm">{t('title')}</h1>
 
         <Button
-          leftIcon={<MdRestartAlt className="text-neon" />}
+          leftIcon={<MdRestartAlt className="text-neon w-6 h-6" />}
           label={commonGeneral('reset')}
-          className="w-fit"
-          variant="text"
+          className="w-fit mr-5"
+          variant="text-slim"
```

Comment:
You can let the variant as "text" and change the `<Clickable />` styles. The `<Base />` has a `h-12`, you can change it to `h-10`